course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- Eigenvalues
+- Elasticity of growth rates
+- Gaussian distribution fitting, mean, standard deviation, probability density function.
+
+### Predator prey population dynamics
+- First order, non-linear, differential equations.
+- Stability analysis, non-trivial equilibrium.
+- deSolve package in R.
+
+### Lectures
+I have put a checkmark on the lessons that seems relevant for the couple of labs that Martin forwarded. Note that this is based on "linguistic similarity" rather than me knowing exactly what topics are included in each lecture... I also don't know to what degree these topics are building on eachother, e.g. do students need to have taken all the previous topics to understand predator prey dynamics? At first glance, it seems like this is _not_ the case, but maybe @mbonsma would know better?
+- [x] Abundance and distribution
+- [x] Population growth
+- [x] Limitation and regulation of populations
+- [ ] Logistic growth
+- [ ] Allee effects 
+- [x] Alternate stable states
+- [ ] Environmental stochasticity
+- [ ] Demographic stochasticity
+- [ ] Stage-structured populations
+- [x] Sensitivity and elasticity
+- [ ] Intro to chaotic population dynamics
+- [ ] Chaos‐tribolium example, time series evidence
+- [ ] Non‐stationarity and critical transitions
+- [ ] Metapopulation dynamics
+- [ ] Source sink dynamics, synchrony, portfolio effects
+- [x] Dispersal, spatial spread, traveling waves
+- [ ] Spread of infectious diseases
+- [ ] Competition (phase portrait, case of coexistence)
+- [ ] Competition (other scenarios)
+- [x] Lotka‐voltera predator‐prey dynamics
+
+
+## The original RQM course
+I have written a short comment under each heading and then pasted from Christie's notes online to easier get an overview of what is included in the original course syllabus.
+
+### Lecture topics
+These are all the topics/bullet points from Christies course, in chronological order. I put a checkmark next to the topics I think are necessary to include, please feel free to add/remove checkmarks, just include comments with the rationale.
+
+- [x] Course syllabus and expectations
+- [x] Open science, open data, & reproducibility
+- [ ] Rules and regulations from funders and institutions
+- [ ] Overview of skills for collaborative science practice
+- [x] Using other people’s data - intro to datasets
+- [x] Spreadsheet best practices
+- [x] Introduction to metadata
+- [ ] Data creation and authorship
+- [x] Introduction to data cleaning
+- [ ] Grey data liberation
+- [x] Introduction to Scripting
+- [x] Data Cleaning in R
+- [x] Version Control
+- [x] R Review
+- [x] The basics of programming in R
+- [x] Even more with R (writing simple software)
+- [ ] Getting a bit abstract, submit to conference
+- [x] Further topics in R
+- [x] Visual communication with data
+- [ ] Ethics and risks in sharing data
+- [x] Plotting in R
+- [ ] Academic publishing, guest speaker
+- [x] Project progress via github task list
```

Comment:
Maybe it could be part of the github lecture if we add that as one of the themes for the currently undetermined weeks.